nixos/modules/services/web-servers/devpi-server.nix

```
@@ -0,0 +1,78 @@
+{ pkgs, lib, config, ... }:
+with lib;
+let
+  serviceName = "devpi-server";
+  cfg = config.services.${serviceName};
+  serverDir = "/var/lib/devpi";
+in {
+  options.services.devpi-server = {
+    enable = mkEnableOption (mdDoc "Devpi Server");
+
+    primaryUrl = mkOption {
+      type = types.str;
```

```suggestion
      type = types.nullOr types.str;
```
This seems to be optional